Automattic/pocket-casts-android
# Introduce contentWithoutConsumedInsets helper

## Description

The idea for this extension function was recommended in a previous PR to make it more transparent that the Compose content is not consuming the insets.

## Testing Instructions
1. On SDK 24
2. Go to Profile -> Settings -> Developer
3. Tap "Show What's New" 
4. ✅ Verify the What's New content doesn't go under the status or navigation bar

## Checklist
 
#### I have tested any UI changes...
